autoload/go/config.vim

```diff
@@ -256,6 +265,22 @@ function! go#config#MetalinterDisabled() abort
   return get(g:, "go_metalinter_disabled", [])
 endfunction
 
+function! go#config#GolangciLintCommand() abort
+  return get(g:, "go_golangci_lint_command", "")
+endfunction
+
+function! go#config#GolangciLintAutosaveEnabled() abort
+  return get(g:, 'go_golangci_lint_autosave_enabled', ['deadcode', 'errcheck', 'ineffassign', 'structcheck', 'typecheck', 'varcheck'])
+endfunction
+
+function! go#config#GolangciLintEnabled() abort
+  return get(g:, "go_golangci_lint_enabled", ['deadcode', 'errcheck', 'ineffassign', 'structcheck', 'typecheck', 'varcheck', 'gosimple', 'govet', 'staticcheck', 'unused'])
+endfunction
+
+function! go#config#GolangciLintDisabled() abort
+  return get(g:, "go_golangci_lint_disabled", ['megacheck'])
```

Comment:
golangci-lint gives a nasty warning message about not being able to run megacheck on code that can't compile unless megacheck is explicitly disabled. there's probably a better way of getting that warning message to go away